doc/commands/transaction.md

```
@@ -0,0 +1,152 @@
+3. [write](#write)
+4. [activate](#activate)
+5. [delete](#delete)
+6. [whereused](#whereused)
+
+## create
+
+Creates a transaction of the given name, description, and package. The
+transaction type must be specified with `-t` and determines which additional
+flags are required.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)